kitsune/sumo/middleware.py

```diff
@@ -222,3 +222,16 @@ def process_response(self, request, response):
             response.set_cookie(mobility.middleware.COOKIE, 'on')
 
         return response
+
+
+class HostnameMiddleware(object):
```

Comment:
I know we don't have to yet, but it would be nice if we started writing middleware in a way that's forward compatible with Django 1.10+, which basically just means having it inherit from `MiddlewareMixin` like I did for django-allow-cidr: https://github.com/mozmeao/django-allow-cidr/blob/master/allow_cidr/middleware.py#L7-L10. Not required, but would be nice to start doing this in prep for the upgrade to 1.11.

Reply:
Nice tip, thanks